packages/table/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+    'react/forbid-prop-types': 'off',
```

Comment:
why are we introducing this?

Reply:
We extend a lot of types from `react-table`, which necessitates the use of `Proptypes.any`

Reply:
Why does it necessitate that?